src/screens/login/components/ClouderyView.js

```diff
@@ -34,7 +38,10 @@ import { rootCozyUrl } from 'cozy-client'
 export const ClouderyView = ({
   clouderyMode,
   setInstanceData,
-  disabledFocus
+  disabledFocus,
+  setError,
+  startOidcOAuth,
+  startOidcOnboarding
```

Comment:
can you update the jsdoc?

Reply:
Fixed in https://github.com/cozy/cozy-flagship-app/pull/700/commits/a3a53a99316ab01c0291d1ac15a8af6c044e3a17